msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -7,49 +7,79 @@ ms.localizationpriority: high
 
-### Show a deep link to an item within your tab
+```javascript
+pages.shareDeepLink({ subPageId: <subPageId>, subPageLabel: <subPageLabel>, subPageWebUrl: <subPageWebUrl> })
+```
+
+You will need to replace the fields with the appropriate information:
+
+* `subPageId`: A unique identifier for the item within your page to which you're deep linking.
+* `subPageLabel`: A label for the item to use for displaying the deep link.
+* `subPageWebUrl`: A fallback URL to use if the client can't render the page.
 
-To show a dialog box that contains a deep link to an item within your tab, call `microsoftTeams.shareDeepLink({ subEntityId: <subEntityId>, subEntityLabel: <subEntityLabel>, subEntityWebUrl: <subEntityWebUrl> })`.
+For more information see, [pages.shareDeepLink()](/javascript/api/@microsoft/teams-js/pages?view=msteams-client-js-latest#@microsoft-teams-js-pages-sharedeeplink&preserve-view=true).
+
+# [TeamsJS v1](#tab/teamsjs-v1)
+
+To implement this, you add a **copy link** action to each item, in whatever way best suits your UI. When the user takes this action, you call `shareDeepLink()` to display a dialog box containing a link that the user can copy to the clipboard. When you make this call, you also pass an ID for your item, which you get back in the [context](~/tabs/how-to/access-teams-context.md) when the link is followed and your tab is reloaded.
+
+```javascript
+microsoftTeams.shareDeepLink({ subEntityId: <subEntityId>, subEntityLabel: <subEntityLabel>, subEntityWebUrl: <subEntityWebUrl> })
+```
 
-Provide the following fields:
+You will need to replace the fields with the appropriate information:
 
 * `subEntityId`: A unique identifier for the item within your tab to which you're deep linking.
 * `subEntityLabel`: A label for the item to use for displaying the deep link.
 * `subEntityWebUrl`: An optional field with a fallback URL to use if the client doesn't support rendering the tab.
 
+---
+
+Alternatively, you can also generate deep links programmatically, using the format specified later in this article. You can use deep links in [bot](~/bots/what-are-bots.md) and [connector](~/webhooks-and-connectors/what-are-webhooks-and-connectors.md) messages that inform users about changes to your tab, or to items within it.
+
+> [!NOTE]
+> This deep link is different from the links provided by the **Copy link to tab** menu item, which just generates a deep link that points to this tab.
+
+>[!IMPORTANT]
+> Currently, shareDeepLink does not work on mobile platforms.
+
 ### Generate a deep link to your tab
```

Comment:
We should probably make it clear that this is really only for back-compatibility in legacy teams. We don't want people thinking they have to craft office specific deep links by hand. If you are looking to start using 2.x you should be using shareDeepLink to generate deep links to your tab.

Reply:
Tried to make it clearer throughout that TeamsJS and the strongly typed APIs were the recommended approach.